jobs/aws/eks-anywhere-build-tooling/hook-presubmit.yaml

```diff
@@ -65,7 +68,7 @@ presubmits:
           runAsUser: 1000
           runAsGroup: 1000
       - name: registry
-        image: registry:2
+        image: public.ecr.aws/docker/library/registry:2
```

Comment:
Is this the default that we're just making the path explicit now?

Reply:
@markapruett the EKS-D presubmit template is set up with an explicit path and I kept it.

Reply:
yes we want to pull from ecr now instead of docker to avoid getting rate limited

Reply:
makes sense.